src/Essentials/src/MainThread/MainThread.uwp.cs

```diff
@@ -16,45 +10,19 @@ static bool PlatformIsMainThread
 		{
 			get
 			{
-#if WINDOWS_UWP
```

Comment:
FYI @mattleibow  I removed this code from essentials because it doesn't seem like it will be valid anymore.